app/views/admin/reports/_status.html.haml

```diff
@@ -35,6 +35,9 @@
       - else
         = fa_visibility_icon(status)
         = t("statuses.visibilities.#{status.visibility}")
+      ·
+      = link_to ActivityPub::TagManager.instance.url_for(status), target: stream_link_target, rel: 'noopener noreferrer', class: 'detailed-status__link' do
+        = t('statuses.view_publicly')
```

Comment:
Whilst this is potentially "unsafe", it's more explicit, though we could also add a warning around this link to ensure moderators know this may be potentially unsafe.